Lab1_Operationalizing_Pytorch_with_Mlflow/src/bert-classifier/news_classifier.py

```diff
@@ -408,5 +408,4 @@ def get_predictions(self, model, data_loader):
             requirements_file="requirements.txt",
             extra_files=["class_mapping.json", "bert_base_uncased_vocab.txt"],
         )
-        mlflow.log_artifacts(args.model_save_path, "state_dict_version")
```

Comment:
Just curious about reasons for this line deletion

Reply:
So I added this initially so that I was able to get the version of the saved model that the handler was expecting (if you go back to [the original repo ](https://github.com/mlflow/mlflow-torchserve/blob/master/examples/BertNewsClassification/news_classifier.py#L387-L394) you'll see that there's no `log_model` or `log_artifacts` commands as they are running this all locally)  
My first solution was to upload the local directory they create using `log_artifacts` (which is the "state_dict_version" you asked about in slack).  
The better solution was to add `extra_files` to the `log_model` command that @michelav had added. Now we know that works, there's no need for the `log_artifacts` command.

Reply:
Another thing worth noting, the "state_dict_version" isn't actually using pytorch's state dict model version, I got confused when I was looking at some open issues on the original repo. The `log_model` command that has the `extra_files` parameter should behave exactly as the `save_model` command [here](https://github.com/mlflow/mlflow-torchserve/blob/master/examples/BertNewsClassification/news_classifier.py#L389)

Reply:
Ah, I see. Thanks for the detailed explanation.